packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.ct.tsx

```diff
@@ -64,3 +64,26 @@ test("should display correctly when disabled", async ({ mount, makeAxeBuilder, p
   // ASSERT
   expect(accessibilityScanResults.violations).toEqual([]);
 });
+
+test("should display correctly when invalid", async ({ mount, makeAxeBuilder, page }) => {
+  // ARRANGE
+  await mount(
+    <OnyxRadioButton
+      id="my-id"
+      label="radio label"
+      name="radio form"
+      value="radio-value"
+      errorMessage="radio error"
+      selected
+    />,
+  );
+
+  // ASSERT
+  await expect(page.locator("input:invalid")).toBeAttached();
```

Comment:
Is there also something like "expect(component).toBeInvalid"?

Reply:
unfortunately not, otherwise I would have used it